04_numeronym/pfannerj/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var mainout io.Writer = os.Stdout
+
+func main() {
+
+	fmt.Fprintln(mainout, numeronyms("accessibility", "Kubernetes", "abc"))
```

Comment:
Little nit, but I don't think the extra white space is required here, please remove.

Reply:
no probs.. will remove...